tuxemon/locale.py

```diff
@@ -247,29 +251,82 @@ def replace_text(session: Session, text: str) -> str:
 
     for i in range(len(player.monsters)):
         monster = player.monsters[i]
-        text = text.replace("${{monster_" + str(i) + "_name}}", monster.name)
-        text = text.replace(
-            "${{monster_" + str(i) + "_desc}}",
-            monster.description,
-        )
-        text = text.replace("${{monster_" + str(i) + "_type}}", monster.slug)
-        text = text.replace(
-            "${{monster_" + str(i) + "_category}}",
-            monster.category,
-        )
-        text = text.replace("${{monster_" + str(i) + "_shape}}", monster.shape)
-        text = text.replace(
-            "${{monster_" + str(i) + "_hp}}",
-            str(monster.current_hp),
-        )
-        text = text.replace(
-            "${{monster_" + str(i) + "_hp_max}}",
-            str(monster.hp),
-        )
-        text = text.replace(
-            "${{monster_" + str(i) + "_level}}",
-            str(monster.level),
-        )
+        # name
```

Comment:
In my opinion, all the comments below are useless.
They are just repetitions of what can already been deduced from reading the name of the variables.